scripts/install_third_party.py

```diff
@@ -338,10 +339,74 @@ def download_manifest_files(filepath):
                     dependency_tar_root_name, dependency_target_root_name)
 
 
+def install_redis_cli():
+    """This installs the redis-cli to the local oppia third_party directory so
+    that development servers and backend tests can make use of a local redis
+    cache. Redis-cli installed here is different from the redis package
+    installed in manifest.json. The redis package detailed in manifest.json is
+    the Python library that allows users to communicate with any Redis cache
+    using Python. The redis-cli package installed in this function contains
+    scripts for the redis-cli and redis-server programs detailed below.
+
+    The Redis-cli is the command line interface that serves up an interpreter
+    that allows users to connects to a redis database cache and query the cache
+    using redis's API.
+
+    Redis-server starts a local Redis database on the machine that can be
+    queried using either the Python redis library or the Redis-cli interpreter.
+    """
+    # We need to install redis-cli separately from the default way since it
```

Comment:
We talked in our meeting about writing more detail to explain how this was different from 3.5.3, right?